open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/pom.xml

```diff
@@ -38,6 +38,11 @@
 	<dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
+    </dependency>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>ui-admin-services-api</artifactId>
+            <version>1.1-SNAPSHOT</version>
```

Comment:
ROgue version here (should be in top level pom)
 - though looks like PR is closed, so just FYI...